nodes/parachain/src/service.rs

```diff
@@ -433,54 +472,63 @@ pub async fn start_parachain_node(
                 telemetry.clone(),
             );
 
-            Ok(AuraConsensus::build::<sp_consensus_aura::sr25519::AuthorityPair, _, _, _, _, _, _>(
-                BuildAuraConsensusParams {
-                    proposer_factory,
-                    create_inherent_data_providers: move |_, (relay_parent, validation_data)| {
-                        let relay_chain_interface = relay_chain_interface.clone();
-                        async move {
-                            let parachain_inherent =
-                                cumulus_primitives_parachain_inherent::ParachainInherentData::create_at(
-                                    relay_parent,
-                                    &relay_chain_interface,
-                                    &validation_data,
-                                    id,
-                                ).await;
-                            let time = sp_timestamp::InherentDataProvider::from_system_time();
-
-                            let slot =
-                                sp_consensus_aura::inherents::InherentDataProvider::from_timestamp_and_duration(
-                                    *time,
-                                    slot_duration.slot_duration(),
-                                );
-
-                            let parachain_inherent = parachain_inherent.ok_or_else(|| {
-                                Box::<dyn std::error::Error + Send + Sync>::from(
-                                    "Failed to create parachain inherent",
-                                )
-                            })?;
-                            Ok((time, slot, parachain_inherent))
-                        }
-                    },
-                    block_import: client.clone(),
-                    para_client: client,
-                    backoff_authoring_blocks: Option::<()>::None,
-                    sync_oracle,
-                    keystore,
-                    force_authoring,
-                    slot_duration,
-                    // We got around 500ms for proposing
-                    block_proposal_slot_portion: SlotProportion::new(1f32 / 24f32),
-                    // And a maximum of 750ms if slots are skipped
-                    max_block_proposal_slot_portion: Some(SlotProportion::new(1f32 / 16f32)),
-                    telemetry,
+            Ok(AuraConsensus::build::<
+                sp_consensus_aura::sr25519::AuthorityPair,
+                _,
+                _,
+                _,
+                _,
+                _,
+                _,
+            >(BuildAuraConsensusParams {
+                proposer_factory,
+                create_inherent_data_providers: move |_, (relay_parent, validation_data)| {
+                    let relay_chain_interface = relay_chain_interface.clone();
+
+                    async move {
+                        let parachain_inherent =
+							cumulus_primitives_parachain_inherent::ParachainInherentData::create_at(
```

Comment:
The formatting here looks odd to me, has the code formatted?

Reply:
Oops, wondering how git action `cargo fmt --check` missed it, looks like tab got used for intend instead of space. corrected it ...

Reply:
Oops, wondering how github action `cargo fmt --all -- --check` is passed.